content/news/playtest-20160930.md

```diff
@@ -0,0 +1,27 @@
+We are pleased to release the second playtest of this development cycle!
+
+Over 5,000 people have tried out the previous playtest, which is about twice as many as usual. Thanks to everyone one of those people for testing and contributing back! Stable releases would not be possible without you! We now count on you to check out this playtest as rigorously as the first one. With any luck, this might be the last chance to report issues before the stable release, so don't miss it!
+
+As usual we have fixed quite a few bugs, many of them very visible and annoying. These include:
+
+* Servers getting unusable in various ways (grey ping, kicking players on joining, etc.).
+* Some hotkeys not working (closing the ingame chat window, navigation hotkeys in the TD mod).
+* Helicopters landing on top of one another.
```

Comment:
Maybe generalize this to something like "more refinement of the aircraft reloading behaviour"? We fixed a lot more than just landing.